docs/framework/configure-apps/file-schema/wcf/customtrackingquery-of-wcf.md

```diff
@@ -19,38 +21,43 @@ Represents a collection of queries that are used to track events that you define
   
 ```xml
 <tracking>
-  <trackingProfile name="Name">
-    <workflow>
-      <customTrackingQueries>
-        <customTrackingQuery activityName="String"
-                             name="String"/>
-      </customTrackingQueries>
-    </workflow>
-  </trackingProfile>
-</tracking>  
+  <profiles>
+    <trackingProfile name="Name">
+      <workflow>
+        <customTrackingQueries>
+          <customTrackingQuery activityName="String"
+                               name="String"/>
+        </customTrackingQueries>
+      </workflow>
+    </trackingProfile>
+  </profiles>
+</tracking>
 ```
 
-## Attributes and Elements  
- The following sections describe attributes, child elements, and parent elements.  
+## Attributes and elements  
+
+The following sections describe attributes, child elements, and parent elements.  
   
 ### Attributes  
   
 |Attribute|Description|  
 |---------------|-----------------|  
-|activityName|A string that specifies the name of the activity that generated the tracking record.|  
-|name|A string that specifies the name of the custom tracking record that is emitted.|  
-  
-### Child Elements  
- None.  
-  
-### Parent Elements  
+|`activityName`|A string that specifies the name of the activity that generated the tracking record.|  
+|`name`|A string that specifies the name of the custom tracking record that is emitted.|  
   
+### Child elements
+
+None.
+
+### Parent elements
+
 |Element|Description|  
 |-------------|-----------------|  
-|[\<customTrackingQuery>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/customtrackingquery.md)|A query that is used to track events that you define in your code activities.|  
+|[\<customTrackingQuery>](customtrackingqueries-of-wcf.md)|Represents a collection of queries that are used to track events that you define in your code activities.|
```

Comment:
The parent is \<customTrackingQueries>. (The link and description are correct, but not the element name.)